tests/events/18-suite-event-mail.t

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test event mail.
+CYLC_TEST_IS_GENERIC=false
```

Comment:
It seems that the `mail` command from the `heirloom-mailx` package is needed.  The (now more common?) `mail` command from GNU `mailutils` doesn't read `smtp` from the environment.